src/server/pipeline/create_master_df.py

```diff
@@ -1,20 +1,19 @@
 import datetime
-
 import pandas as pd
 
 from flask import current_app
+from models import Base
 
 from models import User
 
 
 def __find_and_add_new_rows(connection, new_rows_dataframe):
-    current_app.logger.info('   - Adding new rows to master table')
-    master_df = pd.read_sql('select * from master', connection)
-    master_df = master_df.merge(new_rows_dataframe, how='outer')
-    master_df['created_date'] = datetime.datetime.now()
-    #todo: replace with ORM insert (currently doesn't use the correct schema)
-    master_df.to_sql('master', connection, index=False, if_exists='replace')
+    current_app.logger.info('   - Adding new rows to master and user_info tables')
 
+    new_rows_dataframe = new_rows_dataframe.where(pd.notnull(new_rows_dataframe), None)
+    new_rows_json = new_rows_dataframe.to_dict(orient='records')
+    master_schema = Base.metadata.tables["master"].insert()
+    connection.execute(master_schema, new_rows_json)
```

Comment:
@sposerina and @JoeIlluminati we'll need to add some logic here to:

1. [Parse out the name+email+source vs. ID fields](https://github.com/CodeForPhilly/paws-data-pipeline/pull/123#issuecomment-695813945)
2. Capture the ID's of the newly added rows in master
3. Create new records in user_info using the name, email, source, and master_id

Reply:
After chatting with @dkelley we came up with a solution of writing a joint table, then splitting it afterwards, since the data is all 1:1 mapping between the two tables for MVP.  That should be a lot simpler and faster to write then digging into the details of how sqlalchemy writes postgres queries with nextval and such.  I'll post the code later this week!